package.json

```diff
@@ -38,13 +38,16 @@
     "dist",
     "src"
   ],
+  "dependencies": {
+    "dragselect": "^2.5.4"
+  },
   "devDependencies": {
-    "@blockly/dev-scripts": "^1.2.26",
-    "@blockly/dev-tools": "^4.0.2",
-    "blockly": "^8.0.4"
+    "@blockly/dev-scripts": "^1.2.28",
+    "@blockly/dev-tools": "^5.0.2",
+    "blockly": "^9.1.0"
   },
   "peerDependencies": {
-    "blockly": "^8.0.4"
+    "blockly": "^9.1.0"
```

Comment:
Do we need this to start at 9.1.0, or can we be more inclusive and start at 9.0.0?

Reply:
Of course we can, just updated that.